src/engine/sidechain/engineshoutcast.h

```diff
@@ -118,6 +125,11 @@ class EngineShoutcast : public QObject, public EncoderCallback, public SideChain
     bool m_protocol_is_icecast2;
     bool m_protocol_is_shoutcast;
     bool m_ogg_dynamic_update;
+    QVector<struct shoutcastCacheObject  *> m_pShoutcastCache;
+    bool m_bThreadQuit;
```

Comment:
Should be volatile since it's assigned across thread boundaries -- otherwise compiler optimizations are free to cache it in a register.